docker-compose.yml

```diff
@@ -10,6 +10,25 @@ services:
       - 4222:4222
       - 8222:8222
 
+  server-alpine:
+    privileged: true
+    tty: true
+    build:
+      context: .
+      dockerfile: bin/server_docker/alpine/Dockerfile
+    depends_on:
+      - broker
+    cap_add:
+      - MKNOD
+      - NET_ADMIN
+    networks:
```

Comment:
You don't need to force use default network. Each docker compose creates and maintains it's own network automatically

Reply:
Removed network injection